config/sync/views.view.catalog.yml

```diff
@@ -447,6 +447,10 @@ display:
         type: normal
         title: Catalog
         menu_name: main
+      cache:
+        type: none
+      defaults:
+        cache: false
```

Comment:
I don't know I'm just asking, why do you need to turn off caching?

Reply:
It wasn't intentional, but I fixed it now (see commit `84c22d8`).